src/runtime_src/xdp/profile/database/dynamic_info/aie_db.h

```diff
@@ -40,6 +40,9 @@ namespace xdp {
 
     std::mutex traceLock; // Protects "traceData" vector
 
+    // This is the amount of AIESample threshold AIEDB stores before it flushes to the disk
+    static constexpr uint64_t sampleThreshold = 100000;
```

Comment:
we might want to add 'ull' to the end of the constant value

Reply:
Why? That seems redundant. Do you mean to replace `std::uint64_t` by `auto`?